ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -480,10 +486,9 @@ private ClassLoadContext createVariablesExecutionContext(
             Set<QuotedIdentifier> requiredImports = new HashSet<>();
             String variableType = importsManager.extractImportsFromType(typeSymbol, requiredImports);
             this.newImports.put(variableName, requiredImports);
-
-            // Create a global variable
-            GlobalVariable globalVariable = new GlobalVariable(variableType, variableName,
-                    isAssignableToAny, qualifiersAndMetadata);
+            boolean isDefinedObject = typeSymbol.typeKind().equals(TypeDescKind.OBJECT);
```

Comment:
Again, no need for `equals()`

Reply:
d3e7d2d557259712428f4f178b8d6bfd67325435